src/js/models/carousel_top.js

```diff
@@ -0,0 +1,24 @@
+import { mat4 } from 'gl-matrix'
+import TreeNode from './tree_node'
+import Cylinder from './cylinder'
+
+export default class CarouselTop extends TreeNode {
+    constructor() {
+        super();
+        this.top = new Cylinder();
+        this.something = new Cylinder();
```

Comment:
`this.chair` puede ser un mejor nombre (por ahora que no tenemos las sillas definidas)